src/components/Application/PageElements.tsx

```diff
@@ -96,26 +97,34 @@ const PageElements: React.FC<PageElementProps> = ({
     return (
       <Form>
         <Segment.Group>
-          {elements.map(({ element, thisReviewLatestResponse, isNewApplicationResponse }) => (
-            <Segment key={`question_${element.id}`}>
-              <Grid columns="equal">
-                <Grid.Column floated="left">
-                  <SummaryViewWrapperNEW {...getSummaryViewProps(element)} />
-                </Grid.Column>
-                <Grid.Column floated="right" textAlign="right">
-                  <ReviewButton
-                    isNewApplicationResponse={isNewApplicationResponse}
-                    reviewResponse={thisReviewLatestResponse as ReviewResponse}
-                    summaryViewProps={getSummaryViewProps(element)}
-                  />
-                </Grid.Column>
-              </Grid>
-              <ReviewResponseComponent
-                reviewResponse={thisReviewLatestResponse as ReviewResponse}
-                summaryViewProps={getSummaryViewProps(element)}
-              />
-            </Segment>
-          ))}
+          {elements.map(
+            ({
+              element,
+              thisReviewLatestResponse,
+              isNewApplicationResponse,
+              latestApplicationResponse,
+            }) => (
+              <Segment key={`question_${element.id}`}>
+                <Grid columns="equal">
+                  <Grid.Column floated="left">
+                    <SummaryViewWrapperNEW {...getSummaryViewProps(element)} />
+                  </Grid.Column>
+                  <Grid.Column floated="right" textAlign="right">
+                    <ReviewButton
+                      isNewApplicationResponse={isNewApplicationResponse}
+                      reviewResponse={thisReviewLatestResponse as ReviewResponse}
+                      summaryViewProps={getSummaryViewProps(element)}
+                    />
+                  </Grid.Column>
+                </Grid>
+                <ReviewResponseComponent
+                  latestApplicationResponse={latestApplicationResponse}
+                  reviewResponse={thisReviewLatestResponse as ReviewResponse}
+                  summaryViewProps={getSummaryViewProps(element)}
+                />
+              </Segment>
+            )
+          )}
```

Comment:
Just a heads-up -- this is going to have some nasty conflicts with Nicole's latest PR (#468)

Reply:
bring them on : )